src/flowTypes.ts

```diff
@@ -403,6 +403,7 @@ export interface SendWhatsAppMsg extends Action {
   action_url: string;
   list_items: WhatsAppListItem[];
   quick_replies: string[];
+  attachments?: string[];
```

Comment:
This should not exist, SendWhatsAppMsg card only support one attachment, and it's attribute is already defined above
Remove this new attribute and handle the single attachment instead of creating this new array